arctic/scripts/arctic_delete_library.py

```diff
@@ -28,7 +28,7 @@ def main():
     if not opts.library:
         parser.error('Must specify the full path of the library e.g. arctic_jblackburn.lib!')
 
-    print "Deleting: %s on mongo %s" % (opts.library, opts.host)
+    print("Deleting: %s on mongo %s" % (opts.library, opts.host))
```

Comment:
Python 2.7 supports print function without the future import but the way it handles arguments is different if you use future import and if you don't use future import.

See using Python 2.7

```python
$ source activate py2
discarding //anaconda/bin from PATH
prepending //anaconda/envs/py2/bin to PATH
$ ipython
Python 2.7.11 |Anaconda 2.4.1 (x86_64)| (default, Dec  6 2015, 18:57:58)
Type "copyright", "credits" or "license" for more information.

IPython 4.0.1 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: print("hello ", 2, "people")
('hello ', 2, 'people')

In [2]: from __future__ import print_function

In [3]: print("hello ", 2, "people")
hello  2 people
```

Using Python 3.4 now

```python
$ source deactivate
discarding //anaconda/envs/py2/bin from PATH
$ ipython
Python 3.4.3 |Anaconda 2.3.0 (x86_64)| (default, Oct 20 2015, 14:27:51)
Type "copyright", "credits" or "license" for more information.

IPython 4.0.1 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: print("hello ", 2, "people")
hello  2 people
```

So I suggest using future import for print function to have same behavior with Python 2 and Python 3 when you are using several arguments for print (this is not the case here)